### PR TITLE
Add spellchecking.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/package.json [hunk 1]
```diff
@@ -204,6 +205,7 @@
     "electron": "2.0.5",
     "electron-download": "git://github.com/keybase/electron-download#keybase-fixes-off-410",
     "electron-packager": "12.1.0",
+    "electron-rebuild": "^1.8.2",
```
Comment (chrisnojima) [inline shared/package.json]: do we need to call this now?

Comment (adamjspooner) [reply shared/package.json]: 🤔Maybe not. I'll drop it for now.

**Comment by adamjspooner** [comment]
@keybase/react-hackers, bump.

File: shared/package.json (hunk 1 above)
Comment (adamjspooner) [reply shared/package.json]: 🤔 Maybe so. I needed to run again after updating Node. Are you able to build this branch without first running `./node_modules/.bin/electron-rebuild`?

**Comment by chrisnojima** [comment]
- [ ] osx build
- [ ] linux build
- [ ] windows build (cc: @zanderz can you try this)

**Comment by adamjspooner** [comment]
I'm able to build on Windows (after installing `node-gyp`'s dependencies), but spell-checking doesn't seem to be working (no red dotted line on misspelled words).

**Comment by adamjspooner** [comment]
@chrisnojima, this is working on all desktop clients now. How do we move forward?

**Comment by adamjspooner** [comment]
(macOS Mojave, Windows 10, Ubuntu 18.10)

File: .circleci/config.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ jobs:
     parallelism: 1
 
     docker:
-    - image: keybaseprivate/circleci-client@sha256:7db854b14509ee934ac86b4de16773ded4c136c9a88aae254dc4e9b42f711aea
+    - image: keybaseprivate/circleci-client@sha256:250686fa12ff8b2b5394a09fcb7c530b69e061954fda2eab37051f90c66daba8
```
Comment (akalin-keybase) [inline .circleci/config.yml]: @adamjspooner I'll let you revert this change

**Comment by adamjspooner** [comment]
@keybase/react-hackers, this is ready for review.

**Comment by adamjspooner** [comment]
I'm closing this per our discussion regarding security.

**Pull request closed without merging**